media_events.md

```diff
@@ -0,0 +1,251 @@
+### Generic media events
```

Comment:
Let's add one more section or introduction where we will include information that each media event has to be encapsulated in 

```json
{
    type: "join",
    data: data
}
```